src/lib/Components/LocationMap/index.tsx

```diff
@@ -16,6 +16,10 @@ const Map = styled(Mapbox.MapView)`
   height: 120;
 `
 
+const StyledSerif = styled(Serif)`
```

Comment:
I think we should use a value from Palette where the line height is smaller like `2t` or `3t` if possible

Reply:
ok ill check that out